src/main/java/com/fundynamic/d2tm/Game.java

```diff
@@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
                 TILE_HEIGHT
         );
 
+        container.setVSync(true);
```

Comment:
You dont see tearing effects?

On Sat, Feb 21, 2015 at 12:17 AM, Arjen van der Ende
<notifications@github.com> wrote:

>> @@ -35,6 +35,8 @@ public void initStatesList(GameContainer container) throws SlickException {
>>                  TILE_HEIGHT
>>          );
>>  
>> +        container.setVSync(true);
> Ok, let's keep it then. And buy a better Macbook, because I have no problems here :smiling_imp: 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/stefanhendriks/dune2themaker4j/pull/42/files#r25111740